platforms/Windows/bundle/installer.wxs

```diff
@@ -0,0 +1,33 @@
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:bal="http://wixtoolset.org/schemas/v4/wxs/bal">
+  <Bundle
+    Name="!(loc.BundleName)"
+    Version="$(var.ProductVersion)"
+    Manufacturer="swift.org"
+    Compressed="$(IsBundleCompressed)"
+    UpgradeCode="710F1827-DA4A-4BF4-BDCE-D5F2D7C0DEF2">
+
+    <!-- This bundle should also upgrade the "old" bundles. -->
+    <RelatedBundle Id="8c75f32a-7bdf-4c61-abf6-c7e1c4b8fbf6" Action="upgrade" />
+    <RelatedBundle Id="151d42d9-8877-4b72-ac62-4695243a35c1" Action="upgrade" />
+
+    <BootstrapperApplication>
+      <bal:WixStandardBootstrapperApplication
+      LicenseUrl="https://www.swift.org/LICENSE.txt"
+      LogoFile="swift.png"
+      SuppressOptionsUI="yes"
+      SuppressRepair="no"
+      Theme="hyperlinkLicense" />
+    </BootstrapperApplication>
+
+    <Chain>
+      <MsiPackage SourceFile="$(MSI_LOCATION)\runtime.msi" />
+      <MsiPackage SourceFile="$(MSI_LOCATION)\bld.msi" />
+      <MsiPackage SourceFile="$(MSI_LOCATION)\cli.msi" />
+      <MsiPackage SourceFile="$(MSI_LOCATION)\dbg.msi" />
+      <MsiPackage SourceFile="$(MSI_LOCATION)\ide.msi" />
+      <MsiPackage SourceFile="$(MSI_LOCATION)\swift-format.msi" />
+      <MsiPackage SourceFile="$(MSI_LOCATION)\swift-inspect.msi" />
```

Comment:
Unfortunately, these are not yet generated in all builds.  If they were, I think that we would want to put `swift-format` into `cli.msi` and `swift-inspect` into `dbg.msi`.  Perhaps we can just merge the entries there with an option to just warn if the file is not present and continuing?

Reply:
We can conditionalize on the architecture. Do you want me to move them now? It's just as easy to conditionalize in the bundle as in a package?

Reply:
I think that we can just move them now as long as they don't break the build.  They are intended to be built always and so rather than increasing the number of MSIs further we can reduce them a bit and possibly reduce the amount of code for this.  The preprocessor and property should avoid the MSI for the single entry (the project references make the MSIs seem even sillier than when they are standalone).

Reply:
Sounds reasonable. I'll wire that up in another commit today.

Reply:
(BTW, build-windows-toolchain.bat doesn't build those packages today, so it's actually going to save the build to move them into existing packages.)